synapse/handlers/space_summary.py

```diff
@@ -344,6 +344,7 @@ async def _build_room_entry(self, room_id: str) -> JsonDict:
                 stats["history_visibility"] == HistoryVisibility.WORLD_READABLE
             ),
             "guest_can_join": stats["guest_access"] == "can_join",
+            "origin_server_ts": create_event.origin_server_ts,
```

Comment:
I'd think `creation_ts` would be a better name here?

Reply:
Yeah, that's possible better although that sounds like a comment for the MSC. 😄

Reply:
I've made this change on the MSC and here!